package.json

```diff
@@ -72,6 +72,7 @@
   "dependencies": {
     "@jsdevtools/ono": "^7.1.3",
     "call-me-maybe": "^1.0.1",
-    "js-yaml": "^3.13.1"
+    "js-yaml": "^3.13.1",
+    "node-fetch": "^2.6.1"
```

Comment:
I'm uncertain about this package, as this is clearly meant to be used in the context of Node.js, while json-schema-ref-parser is also supposed to work in modern browsers.
Although I know we used `http` and `https` Node modules in the existing codebase, I suggest we use [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch).
Most of robust bundlers such as Webpack or Rollup were able to shim Node moduels such as `http` and `https` and doing so might be potentially slightly more tricky if we rely on external libraries (that may change their implementation any time).